utils/gh-update.py

```diff
@@ -97,15 +98,15 @@ def update_status(repo, commit, token, data):
     api_url = ("https://api.github.com/repos/%s/statuses/%s" %
                (repo, commit))
 
-    print "Updating status of commit", commit, "with data", data
+    print("Updating status of commit", commit, "with data", data)
```

Comment:
I think then we need to add:

```python
from __future__ import print_function
```

at the top. Or it'll print like a tuple on Python 2.